docs/07-theme-developer-tutorial/06-js-api.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Theme Developer Tutorial: 6. Using the JS API"
+short_title: 6 - JS API
+id: theme-developer-tutorial-js-api
+---
+
+In the last couple of chapters, we've explored how to use the JavaScript API to render content into outlets. `renderInOutlet` is the most commonly-used API, but there are a ton more! In this chapter we'll try out a few of them, and show you how to discover more.
+
+## Common API methods
+
+### getCurrentUser()
+
+`api.getCurrentUser()` will return information about the current user, or `null` if nobody is logged in. This can be used for all sorts of things, including per-group logic, or rendering a user's username into the UI.
+
+```gjs
+import { apiInitializer } from "discourse/lib/api";
+
+export default apiInitializer((api) => {
+  console.log("Current user is", api.getCurrentUser());
+});
+```
+
+### headerIcons
+
+`api.headerIcons` will allow you to add, remove and re-arrange icons in the header. This is one of our ["DAG"](https://en.wikipedia.org/wiki/Directed_acyclic_graph) APIs, which allows you to control the ordering of things via "before" and "after" parameters.
```

Comment:
```suggestion
`api.headerIcons` will allow you to add, remove and re-arrange icons in the header. This is one of our ["DAG"](https://en.wikipedia.org/wiki/Directed_acyclic_graph) APIs, which allows you to control the ordering of things via "before" and "after" parameters.
```

Minor, but IDK if it's relevant to explain that this is a DAG, the wikipedia article we link to here is gnarly

Reply:
Yeah fair point. Removing